cohere_sagemaker/client.py

```diff
@@ -174,5 +316,91 @@ def rerank(self,
         
         return reranking
 
-    def close(self):
+    def classify(self, input: List[str], name: str) -> Classifications:
+        json_params = {"texts": input, "model_id": name}
+        json_body = json.dumps(json_params)
+
+        params = {
+            "EndpointName": self._endpoint_name,
+            "ContentType": "application/json",
+            "Body": json_body,
+        }
+
+        try:
+            result = self._client.invoke_endpoint(**params)
+            response = json.loads(result["Body"].read().decode())
+        except EndpointConnectionError as e:
+            raise CohereError(e)
```

Comment:
Just realized that this is not correct because `e` is not a string. It was a copy paste from the code in generate() which is also wrong. Will fix it tomorrow morning.
What's the best option here? I'm not sure why we'd want to add a wrapper around the error, that just makes the error trace more complicated

Reply:
After talking with Jeremy, I decided to keep the `CohereError` and replace `e` by `str(e)` so that it doesn't error out.